docs/Usage-RASP.md

```diff
@@ -132,6 +132,26 @@ if (Platform.OS == 'android') {
 }
 ```
 
+## Advanced protection against Tapjacking
+
+By default, when Malwarelytics for React-Native is initialized on Android, it automatically protects the current application's activity against tapjacking and all activities created after initialization. If your application created other `Activity` objects before the Malwarelytics module was initialized, you will need to register them for protection manually. For example:
+
+```java
+import com.wultra.android.malwarelytics.reactnative.MalwarelyticsNativeHelper;
+
+public class AnotherActivity extends ReactActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        MalwarelyticsNativeHelper.getInstance().registerActivityForTapjackingProtection(this);
+    }
+}
+```
+
+<!-- begin box info -->
+You can read more about Tapjacking protection in [Android Native Library's](https://github.com/wultra/antivirus/blob/develop/docs/Tapjacking-Protection.md) documentation.
+<!-- end -->
```

Comment:
Also add link to https://developers.wultra.com/components/malwarelytics-android/develop/documentation/Activity-Protection

Reply:
I see. But there's a bit of inconvenience for folks that check the docs on the github repo instead of our developer portal. They will see an invalid link pointing to a private repo.